lib/SPIRV/libSPIRV/spirv_internal.hpp

```diff
@@ -104,6 +105,7 @@ enum InternalCapability {
   ICapabilityTensorFloat32RoundingINTEL = 6425,
   ICapabilityMaskedGatherScatterINTEL = 6427,
   ICapabilityJointMatrixWIInstructionsINTEL = 6435,
+  ICapabilityCooperativeMatrixInvocationInstructionsINTEL = 6435,
```

Comment:
May be this question belongs elsewhere. But I will ask here. How do we make sure that the tokens ppl use in Khronos registry do not overlap with these?

Thanks

Reply:
Vendors reserve the range of tokens for themselves in https://github.com/KhronosGroup/SPIRV-Headers/blob/main/include/spirv/spir-v.xml . It's up for a particular vendor to setup an internal process to distribute these tokens among vendor extensions.